domain/src/main/java/com/playone/mobile/domain/model/Playone.kt

```diff
@@ -1,22 +1,36 @@
 package com.playone.mobile.domain.model
 
+import java.util.Date
 import kotlin.properties.Delegates
 
 /**
  * Representation for a [Playone] fetched from an external layer data source
  */
-class Playone {
+sealed class Playone {
 
-    lateinit var id: String
-    lateinit var name: String
-    lateinit var description: String
-    var date by Delegates.notNull<Long>()
-    var updated by Delegates.notNull<Long>()
-    lateinit var address: String
-    var longitude by Delegates.notNull<Double>()
-    var latitude by Delegates.notNull<Double>()
-    var limit by Delegates.notNull<Int>()
-    var level by Delegates.notNull<Int>()
-    lateinit var host: String
-    lateinit var userId: String
+    class Detail : Playone() {
+        lateinit var id: String
+        lateinit var name: String
+        lateinit var description: String
+        var date by Delegates.notNull<Long>()
+        var updated by Delegates.notNull<Long>()
+        lateinit var address: String
+        var longitude by Delegates.notNull<Double>()
+        var latitude by Delegates.notNull<Double>()
+        var limit by Delegates.notNull<Int>()
+        var level by Delegates.notNull<Int>()
+        lateinit var host: String
+        lateinit var userId: String
+    }
+
+    data class CreateParameters(
+        var name: String,
+        var description: String,
+        var playoneDate: Date,
+        val longitude: Long,
+        var latitude: Long,
+        var address: String,
+        var limitPeople: Int,
```

Comment:
Why don't we use only one data class(Entity class)? Maintaining two class will cause inconsistent data class.

Reply:
Also, our Entity Class will be a normal class, right? Do I miss something!?

Reply:
@pokk If use only one class it must have a field that is nullable i.e., `id` when creating Playone haven't had ID at the moment. That is why I made two class and one of two is not a entity class it is for parameters

Reply:
`limitPeople` should be in the `Detail` class I think. 
We're able to avoid to as like kinda issues we missed some variables without the intent.